packages/at_client/lib/src/client/at_client_impl.dart

```diff
@@ -68,7 +70,8 @@ class AtClientImpl implements AtClient {
       {AtClientManager? atClientManager,
       RemoteSecondary? remoteSecondary,
       EncryptionService? encryptionService,
-      SecondaryKeyStore? localSecondaryKeyStore}) async {
+      SecondaryKeyStore? localSecondaryKeyStore,
+      AtChops? atChops}) async {
```

Comment:
You need to add atChops parameter to the private constructor, and pass this atChops parameter to it.

Reply:
done